docs/DeveloperGuide.adoc

```diff
@@ -79,15 +81,38 @@ image::UiClassDiagram.png[]
 
 *API* : link:{repoURL}/src/main/java/seedu/address/ui/Ui.java[`Ui.java`]
 
-The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
+The UI consists of a `MainWindow` that is made up of parts e.g.`BrandingLabel`, `TaskListPanel`,
```

Comment:
Consider changing the name of the classes at least to make them consistent with your application (from `Person` to `Task`)